test/fake_device_factory_test.rb

```diff
@@ -181,11 +182,9 @@
     io = StringIO.new(input.join("\n"))
     Y2Storage::FakeDeviceFactory.load_yaml_file(staging, io)
 
-    expect(staging.num_devices).to eq 7
-    expect(staging.num_holders).to eq 5
-    enc = Storage.to_encryption(Storage::BlkDevice.find_by_name(staging, "/dev/mapper/cr_root"))
-    expect(enc.name).to eq "/dev/mapper/cr_root"
-    expect(enc.password).to eq "s3cr3t"
+    root_part = Storage.to_partition(Storage::BlkDevice.find_by_name(staging, "/dev/sda2"))
+    expect(root_part.encryption.name).to eq "/dev/mapper/cr_root"
+    expect(root_part.encryption.filesystem.mountpoints.first).to eq "/"
```

Comment:
No need to remove the check for `enc.password`. Don't take my previous suggestion too literally. I just wanted to make sure the test checked the relationship between the Luks and Partition objects. Didn't mean to replace the whole test or to stop using the `enc` variable for readability.